operations/mimir-mixin/dashboards/tenants.libsonnet

```
@@ -530,5 +530,106 @@ local filename = 'mimir-tenants.json';
         ) +
         { legend: { show: false } },
       )
-    ),
+    )
+
+    .addRow(
+      $.row('Read Path - Query Frontend Insights')
```

[nit] The official name is "query-frontend". **Please replace "Query Frontend" in the rest of the file too**.

```suggestion
      $.row('Read Path - Query-frontend Insights')
```